jobs/build.Jenkinsfile

```
@@ -228,11 +228,34 @@ lock(resource: "build-${params.STREAM}") {
             new_version = shwrapCapture("/usr/lib/coreos-assembler/fcos-versionary")
         }
 
+        def overrides_fetch_param = ""
+        def recent_commits_to_lockfiles = shwrapRc('''
+            # Temporary assignment to a particular file:
+            cd ./src/config/
+            echo "Last commit: $(date -ud @$(git log -1 --format="%ct" manifest-lock.*))"
+            # Number of minutes to check
+            n=120
+            current_time=$(date +%s)
+            minutes_ago=$(( current_time - ( 60 * n ) ))
+            last_log=$(git log -1 --format="%ct" manifest-lock.*)
+            cd ../../
+            # Check if git log was run within the last n minutes
+            if (( last_log > minutes_ago )); then
+                echo "Last log < $n minutes; implementing --with-cosa-overrides"
+                python3 download-overrides.py
+                exit 0
+            else
+                exit 1
+            fi
+        ''')
+
         // fetch from repos for the current build
         stage('Fetch') {
-            shwrap("""
-            cosa fetch ${strict_build_param}
-            """)
+            // Dont run this for production builds
+            if (recent_commits_to_lockfiles == 0 && stream_info.type != "production" ) {
```

I think you need `()` since it's a function call:

```suggestion
            if (recent_commits_to_lockfiles() == 0 && stream_info.type != "production" ) {
```